seed-db/scripts/index.js

```diff
@@ -15,10 +18,13 @@ const argv = yargs
     .help()
     .alias("help", "h").argv;
 
+function writeFile (filename,obj) {
+    const jsonObj = JSON.stringify(obj);
+    fs.writeFileSync(filename, jsonObj);
+}
+
```

Comment:
```suggestion
const writeFile = (filename,obj) => {
    const jsonObj = JSON.stringify(obj);
    fs.writeFileSync(filename, jsonObj);
}

```

Could we use es6 functions to keep up with the convention used throughout the application please